Foods.yaml

```diff
@@ -6,5 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["Pizza"]
-  
\ No newline at end of file
+  - ["Pizza", "kale_smoothies"]
```

Comment:
Why are these different styles? We doing camel case or snake case?

Reply:
We should try to stick to `snake_case` for yaml files in the NUbots codebase(s)!

Reply:
good point!